samples/internat/readme.txt

```diff
@@ -24,74 +24,54 @@ Q. Why the message when I enter '17' is only partly translated?
 A. This will only work under some versions of Linux, don't worry if the second
    half of the sentence is not translated.
 
-Q. I don't speak french, what about translations to <language>?
-A. Please write them - see the next question. French is chosen by default
-   because it's the only translation which is currently available. To test
-   translations to the other languages please run the sample with 2 command line
-   arguments: the full language name and the name of the directory where the
-   message catalogs for this language are (will be taken as 2 first letters of
-   the language name if only 1 argument is given).
-
 Q. How to do translations to other language?
-A. First of all, you will need the GNU gettext tools (see the next questions).
+A. First of all, you will need the GNU gettext tools.
```

Comment:
Since poedit is much nicer to work with for people used to GUI tools, I would add 
```suggestion
A. First of all, you will need the GNU gettext tools or, if you prefer GUI applications, a program such as poedit.
```
This would probably be better done somewhere earlier in the readme, but the alternative to gettext tools should be mentioned somewhere. Assuming wxWidgets sample readme is a good place for teaching people this stuff...

Reply:
Yes, I agree that it would be worth mentioning Poedit.

As for doing it in this README -- the best would probably be to move its entire contents into i18n overview and add a comment to the sample pointing to it.